packages/editor/src/components/default-block-appender/style.scss

```diff
@@ -1,14 +1,16 @@
 .editor-default-block-appender {
-	input[type="text"].editor-default-block-appender__content { // needs specificity
+	clear: both; // The appender doesn't scale well to sit next to floats, so clear them.
+
+	input[type="text"].editor-default-block-appender__content { // Needs specificity in order to override upstream input field styles.
```

Comment:
Single question: is this upstream in Gutenberg, or in WordPress Core? Would be handy to mention; it made me curious.

Reply:
Clarified further, will merge when the checks pass. It's WordPress core :)